.github/workflows/universal_tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Run universal test suite
+
+on: [pull_request]
+
+jobs:
+  universal-tests:
+    runs-on: [ubuntu-latest]
+    steps:
```

Comment:
Eventually, do we want to test the feast trino code on multiple version of Python?

Reply:
Yes https://github.com/Shopify/feast-trino/issues/21